CODESTYLE.md

```diff
@@ -0,0 +1,129 @@
+# Coding style guide
+
+
+## Objective-C
+We follow the [Google Objective-C Style Guide](https://google.github.io/styleguide/objcguide.html).
+Make sure your code is also follows [Cocoa Coding Guidelines](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/CodingGuidelines.html).
+
+We use automation tools to enforce formatting (see `Enforcing formatting and linting` section below).
+
+## Swift
+
+We follow the [Google Swift Style Guide](https://google.github.io/swift/)
+Please note that [Apple's Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/) are considered as a part of the style guide.
+
+There're following items that can differ a bit from Style Guide above:
+- We do NOT have limit on line length, but it's recommended to have lines less than 100 characters
```

Comment:
Let's add preferred line length without enforcing it. 120 characters would be a good guide.

Reply:
No issues with that, updated.